### PR TITLE
libvirt_xml: Fix pylint E1003

### PR DESCRIPTION
**Pull request by waynesun09**
Fix E1003 with replace self.__class__ with the current class

Signed-off-by: Wayne Sun <gsun@redhat.com>

### CODE REVIEW

**Comment by lmr** [comment]
@waynesun09 Please rebase it and create a comment removing the E1003 from the .travis-ci.yaml file `inspekt` command `--disable-lint` flag. Once the CI passes, I'll gladly merge this.

File: virttest/libvirt_xml/devices/disk.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def __init__(self, virsh_instance=base.base.virsh):
                                    tag_name='config', attribute='file')
             accessors.XMLAttribute('snapshot_name', self, parent_xpath='/',
                                    tag_name='snapshot', attribute='name')
-            super(self.__class__, self).__init__(virsh_instance=virsh_instance)
+            super(Disk.DiskSource, self).__init__(virsh_instance=virsh_instance)
```
Comment (lmr) [inline virttest/libvirt_xml/devices/disk.py]: Wait... Why Disk.DiskSource?

Comment (lmr) [inline virttest/libvirt_xml/devices/disk.py]: Ah, nested classes. I forgot about those, never liked nested classes...

**Comment by lmr** [comment]
@waynesun09, So, for now I just ignore E1003. If you can rebase it, add the extra commit removing the ignore list in Travis, and test it it'd be great.

File: virttest/libvirt_xml/devices/disk.py (hunk 1 above)
Comment (waynesun09) [inline virttest/libvirt_xml/devices/disk.py]: Yeah...

**Comment by lmr** [comment]
Thanks @waynesun09 !

**Pull request merged**